Description: Ignore the JMH plugin
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.gradle
+++ b/build.gradle
@@ -35,7 +35,6 @@
   }
 }
 
-plugins { id 'me.champeau.jmh' version '0.6.6' apply false }
 plugins { id 'org.sonarqube' version '3.3' apply false }
 
 description = 'ASM, a very small and fast Java bytecode manipulation framework'
@@ -118,10 +117,8 @@
 // Use "gradle benchmarks:jmh [-PjmhInclude='<regex>']" to run the benchmarks.
 project(':benchmarks') {
   description = "Benchmarks for ${rootProject.description}"
-  apply plugin: 'me.champeau.jmh'
   dependencies {
     implementation files('libs/csg-bytecode-1.0.0.jar', 'libs/jclasslib.jar')
-    jmh project(':asm'), project(':asm-tree') 
   }
   depends = [
     'kawa:kawa:1.7',
@@ -145,12 +142,12 @@
     classes.dependsOn "asm${version}"
   }
   configurations.create('input-classes-java8')
-  dependencies.add('input-classes-java8', 'io.vavr:vavr:0.10.0@jar')
   task copyInputClasses(type: Copy) {
     from configurations.'input-classes-java8'.collect { zipTree(it) }
     into "${buildDir}/input-classes-java8"
   }
   classes.dependsOn copyInputClasses
+/*
   jmh {
     jvmArgsAppend = ["-Duser.dir=${rootDir}"]
     resultFormat = 'CSV'
@@ -159,6 +156,7 @@
       include = [jmhInclude]
     }
   }
+*/
 }
 
 project(':tools') {
